utils/middlewares/userDataProcessing.ts

```diff
@@ -0,0 +1,26 @@
+import { UserDataProcessingChoiceRequest } from "io-functions-commons/dist/generated/definitions/UserDataProcessingChoiceRequest";
+import {
+  NewUserDataProcessing,
+  UserDataProcessing
+} from "io-functions-commons/dist/src/models/user_data_processing";
+import { RequiredBodyPayloadMiddleware } from "io-functions-commons/dist/src/utils/middlewares/required_body_payload";
+/**
+ * A middleware that extracts a NewUserDataProcessing payload from a request.
+ */
+export const NewUserDataProcessingMiddleware = RequiredBodyPayloadMiddleware(
```

Comment:
can we use this inline in handler.ts ?

Reply:
I suppose that we can't, because in request payload we have the object wrapper of a UserDataProcessingChoice ( we use UserDataProcessingChoiceRequestMiddleware for this purpose ).